front/src/modules/ui/menu-item/components/MenuItemMultiSelectAvatar.tsx

```diff
@@ -0,0 +1,51 @@
+import { ReactNode } from 'react';
```

Comment:
we should add stories on this one

Reply:
Created a ticket for this : https://github.com/twentyhq/twenty/issues/1464